glusterd2/device/devicereq.go

```diff
@@ -0,0 +1,9 @@
+package device
+
+import "github.com/pborman/uuid"
+
+// AddDeviceReq structure
+type AddDeviceReq struct {
+	PeerID uuid.UUID `json:"peerid"`
```

Comment:
Name the json string as `peer-id` to be consistent with existing naming convention